andrewinsoul/RoleManagementSystem
# Seed data

#### What does this PR do?
It seeds data into the database

#### Description of Task to be completed?
- Create npm script that seeds data into the database

#### How should this be manually tested?
- Fetch the branch locally
- cd into the server folder
- Run npm migrate to create the database tables
- Run npm seed and your database should be populated
